Source/ASDisplayNode+Layout.mm

```diff
@@ -654,10 +653,10 @@ - (void)transitionLayoutWithSizeRange:(ASSizeRange)constrainedSize
         
         // Update calculated layout
         let previousLayout = _calculatedDisplayNodeLayout;
-        let pendingLayout = std::make_shared<ASDisplayNodeLayout>(newLayout,
-                                                                  constrainedSize,
-                                                                  constrainedSize.max,
-                                                                  newLayoutVersion);
+        var pendingLayout = ASDisplayNodeLayout(newLayout,
```

Comment:
Do you change the `pendingLayout` somewhere to change it to `var`?

Reply:
You're right – in a previous version, `setCalculatedLayout` mutated the input (to apply the unflattenedLayout thing) but I changed it to copy the input and never committed reverting this to `let`.